lib/spack/spack/util/web.py

```diff
@@ -70,11 +69,11 @@ def custom_ssl_certs() -> Optional[Tuple[bool, str]]:
     if not ssl_certs:
         return None
     path = spack.util.path.substitute_path_variables(ssl_certs)
-    if not os.path.isabs(path):
+    if not PurePath(path).is_absolute():
```

Comment:
This is one of the cases where `spack.util.path.substitute_path_variables` will eventually be pathlib aware. I think we should, particularly in that sort of case, cast the call to `spack.util.path.subs....` directly, so that way the returned object is a Pathlib object and this method can be refactored as if `spack.util.path` returned pathlib objects.